Mathlib/SetTheory/Ordinal/CantorNormalForm.lean

```diff
@@ -55,6 +57,28 @@ theorem CNFRec_pos (b : Ordinal) {o : Ordinal} {C : Ordinal → Sort*} (ho : o 
     CNFRec b H0 H o = H o ho (@CNFRec b C H0 H _) := by
   rw [CNFRec, dif_neg]
 
+/-- Inducts on the base `b` expansion of an ordinal, for `1 < b ≤ ω`.
+
+This differs from `CNFRec` in that every instance of `b ^ a` is considered separately. -/
+@[elab_as_elim]
+noncomputable def CNFRec' {C : Ordinal → Sort*} {b : Ordinal} (hb₁ : 1 < b) (hb : b ≤ ω) (H0 : C 0)
```

Comment:
Per the naming conventions, this and the def above should be `cnfrec` or perhaps `cnfRec` or `recCNF`. To avoid deprecating your new name as soon as its introduced, can you make a PR to fix the existing name first?